main.js

```diff
@@ -27,23 +30,105 @@ class ModuleInstance extends InstanceBase {
 		this.config = config
 	}
 
+	async sendCam(self, str) {
+		const { default: got } = await import('got');
+		if (str) {
+
+			const url = `https://${self.config.httpHost}:${self.config.httpPort}${str}`
+
+			if (self.config.debug) {
+				self.log('debug', `Sending : ${url}`)
+			}
+
+			try {
+				const response = await got.put(
+					url,
+					{
+						username:self.config.username,
+						password:self.config.password,
+						https: { rejectUnauthorized: false }
+					}
+				)
+				console.log("Result from REST:" + str);
+			} catch (err) {
+				throw new Error(`Action failed err: ${err}`)
```

Comment:
Suggest to reference which action failed and do a more user oriented error handling, e.g. setting status and logging error message.